swag/swag_client.py

```diff
@@ -369,3 +416,335 @@ async def execute_swagdmin_command(self, message):
+                .cagnotte_currency
+            )
+            cagnotte_name = (
+                self.swag_bank.get_active_cagnotte(cagnotte_idx)
+                .get_info()
+                .cagnotte_name
+            )
+
+            await message.channel.send(
+                f"{message.guild.get_member(gagnant).mention} vient de gagner l'intégralité de la €agnotte "
+                f"€{cagnotte_idx} *{cagnotte_name}*, à savoir `{gain} {cagnotte_currency}` ! 🎰"
+            )
+
+            await update_forbes_classement(message.guild, self, self.client)
+
+        elif "détruire" in splited_command:
+            cagnotte_idx = get_cagnotte_idx_from_command(splited_command)
+            cagnotte_name = (
+                self.swag_bank.get_active_cagnotte(cagnotte_idx)
+                .get_info()
+                .cagnotte_name
+            )
+
+            self.swag_bank.detruire_cagnotte(cagnotte_idx, message.author.id)
+            await message.channel.send(
+                f"La €agnotte €{cagnotte_idx} *{cagnotte_name}* est maintenant détruite de ce plan de l'existance ❌"
+            )
+            await update_forbes_classement(message.guild, self, self.client)
+
+        else:
+            await message.channel.send(
+                f"{message.author.mention}, tu as l'air perdu "
+                "(pourquoi ça ne m'étonne pas...) 🙄\nVoici les commandes "
+                "que tu peux utiliser avec les €agnottes :\n"
+                "```HTTP\n"
+                "!€agnotte créer [$wag/$tyle] [Nom_de_la_€agnotte] ~~ "
+                "Permet de créer une nouvelle €agnotte, de $wag ou de $tyle "
+                "avec le nom de son choix\n"
+                "!€agnotte info €[n] ~~ Affiche des informations détaillés sur la €agnotte n\n"
+                "!€agnotte historique €[n] ~~ Affiche les transactions en lien avec la €agnotte n\n"
+                "!€agnotte payer €[n] [montant] ~~ fait don "
+                "de la somme choisi à la €agnotte numéro €n\n"
+                "⭐!€agnotte donner €[n] [montant] [@mention] ~~ donne à l'utilisateur mentionné "
+                "un montant venant de la cagnotte\n"
+                "⭐!€agnotte partager €[n] [@mention1 @mention2 ...] ~~ "
+                "Partage l'intégralité de la €agnotte entre les utilisateurs mentionné. "
+                "Si personne n'est mentionné, la €agnotte sera redistribué parmis ses donateurs\n"
+                "⭐!€agnotte loto €[n] [@mention1 @mention2 ...] ~~ "
+                "Tire au sort parmis les utilisateurs mentionnés celui qui remportera l'intégralité "
+                "de la €agnotte. Si personne n'est mentionné, le tirage au sort se fait parmis"
+                "l'ensemble des personnes ayant un compte\n"
+                "⭐!€agnotte détruire €[n] ~~ Détruit la €agnotte si elle est vide"
+                "```\n"
+                "*Seul le créateur de le Gestionnaire de la €agnotte peut faire les commandes précédées d'une  ⭐*"
```

Comment:
seul le créateur de le Gestionnaire ?

Reply:
Resolved in https://github.com/BobbySoftwares/Bybbot/pull/7/commits/173d6bc07cf9a1a3121d5820f16d9bcb19a74a16